pygfx/linalg/matrix4.py

```diff
@@ -967,7 +967,7 @@ def make_orthographic(
         te[13] = -y
         te[2] = 0
         te[6] = 0
-        te[10] = -2 * p
+        te[10] = -1 * p
         te[14] = -z
         te[3] = 0
         te[7] = 0
```

Comment:
I'm making changes here to lines that probably came from somewhere else. I see that vispy uses the same. So either the original source is wrong, or this should be fixed in the OrthographicCamera class, by massaging `far`  and `near` and multiplying the resulting matrix with a translation in `z`.

Reply:
I think it makes sense to *not* treat z the same as x and y, because `x` and `y` in NDC range between 0 and 1, while the depth is expressed in values between 0.0 and 1.0.

Reply:
It was directly ported from threejs, so I'm hesitant to think they have a bug..

Reply:
Ok, I'll do some more digging. Perhaps `make_orthographic` is supposed to produce such a matrix, and we must fix it a up for the `[0..1]` range in the camera class then.